.vscode/settings.json

```diff
@@ -3,7 +3,7 @@
   "editor.formatOnSave": true,
   "editor.defaultFormatter": "esbenp.prettier-vscode",
   "editor.codeActionsOnSave": {
-    "source.fixAll.eslint": true,
-    "source.fixAll.format": true
+    "source.fixAll.eslint": "explicit",
+    "source.fixAll.format": "explicit"
```

Comment:
Can you explain why this is needed?

Reply:
Oops, didn't mean to include that! I was running into issues where saving would cause weird auto-formatting issues with the math formulas.